magenta/tools/magenta-install.sh

```diff
@@ -51,13 +64,14 @@ if [[ ! $(which conda) ]]; then
     echo "==========================================="
     echo ""
     readonly CONDA_INSTALL="/tmp/${MINICONDA_SCRIPT}"
+    readonly CONDA_PREFIX="${HOME}/miniconda2"
     curl "https://repo.continuum.io/miniconda/${MINICONDA_SCRIPT}" > "${CONDA_INSTALL}"
-    bash $CONDA_INSTALL
-    # Miniconda installer appends to path differently for different OS
-    if [[ $OS == "LINUX" ]]; then
-        source "${HOME}/.bashrc"
-    elif [[ $OS == "MAC" ]]; then
-        source "${HOME}/.bash_profile"
+    bash "${CONDA_INSTALL}" -p "${CONDA_PREFIX}"
+    # Modify the path manually rather than sourcing .bashrc because some .bashrc
+    # files refuse to execute if run in a non-interactive environment.
+    export PATH="${CONDA_PREFIX}/bin:${PATH}"
+    if [[ ! $(which conda) ]]; then
+      err 'Could not find conda command'
```

Comment:
You could make this a little bit more instructive in this context. maybe something like  'Could not find conda command. Conda binary not properly added to PATH'